tests/server_send.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace ioremap::elliptics;
 using namespace boost::unit_test;
 
+#define ENABLE_LONG_TESTS 0
```

Comment:
It means this test will never run, should we at least provide cmake config option?
How long does it take to run this test, maybe we should turn it on always?